web/src/features/fireWeather/components/maps/WeatherMap.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState, useCallback } from 'react'
+import { fromLonLat, get } from 'ol/proj'
+import * as olSource from 'ol/source'
+import GeoJSON from 'ol/format/GeoJSON'
+import { Circle as CircleStyle, Fill, Stroke, Style } from 'ol/style'
+import { FeatureLike } from 'ol/Feature'
+
+import WeatherStationsGeoJson from 'features/fireWeather/components/maps/weather-stations-geojson.json'
+import Map from 'features/map/Map'
+import TileLayer from 'features/map/TileLayer'
+import VectorLayer from 'features/map/VectorLayer'
+import LayerSwitch from 'features/fireWeather/components/maps/LayerSwitch'
+
+const styles = {
+  Point: new Style({
+    image: new CircleStyle({
+      radius: 4,
+      fill: new Fill({
+        color: '#E59982'
+      }),
+      stroke: new Stroke({
+        color: 'black'
+      })
+    })
+  })
+}
+
+const getArcGISAttribution = (service: string): string =>
+  `Tiles © <a href="https://services.arcgisonline.com/ArcGIS/rest/services/${service}/MapServer" target="_blank">ArcGIS</a>`
+const getArcGISMapUrl = (service: string): string =>
+  `https://server.arcgisonline.com/ArcGIS/rest/services/${service}/MapServer/tile/{z}/{y}/{x}`
```

Comment:
I don't think we're licensed to use this data - see:  https://developers.arcgis.com/documentation/mapping-apis-and-location-services/licensing/license-location-services/